gatsby-node.js

```diff
@@ -84,6 +85,22 @@ const constructDbFilter = () => ({
     commit_hash: process.env.COMMIT_HASH || { $exists: false },
 });
 
+const getRelatedPagesWithImages = pageNodes => {
+    const related = dlv(pageNodes, 'query_fields.related', []);
+    let relatedPageInfo = [];
```

Comment:
nit: since we're guaranteeing an array for `related` we can just return the below map without checking for length since the map will return and empty array when run on an empty array  - e.g. `return related.map(<...>)`

This assumes that `'`query_fields.related` would be undefined and not null.